docs/usage.md

```diff
@@ -1,153 +1,306 @@
 # nf-core/raredisease: Usage
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/raredisease/usage](https://nf-co.re/raredisease/usage)
+**We recommend reading this documentation on the nf-core website: [https://nf-co.re/raredisease/usage](https://nf-co.re/raredisease/usage)**
+
+Table of contents:
+
+- [nf-core/raredisease: Usage](#nf-core-raredisease--usage)
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Run nf-core/raredisease with test data](#run-nf-core-raredisease-with-test-data)
+- [Run nf-core/raredisease with your data](#run-nf-core-raredisease-with-your-data)
+  - [Samplesheet](#samplesheet)
+  - [Reference files and parameters](#reference-files-and-parameters)
+    - [1. Alignment](#1-alignment)
+    - [2. QC stats from the alignment files](#2-qc-stats-from-the-alignment-files)
+    - [3. Repeat expansions](#3-repeat-expansions)
+    - [4. Variant calling - SNV](#4-variant-calling---snv)
+    - [5. Variant calling - Structural variants](#5-variant-calling---structural-variants)
+    - [6. SNV annotation & Ranking](#6-snv-annotation---ranking)
+    - [7. SV annotation & Ranking](#7-sv-annotation---ranking)
+    - [8. Mitochondrial analysis](#8-mitochondrial-analysis)
+  - [Run the pipeline](#run-the-pipeline)
+    - [Direct input in cli](#direct-input-in-cli)
+    - [Import from a config file (recommended)](#import-from-a-config-file--recommended-)
+- [Best practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+  - [Resource errors](#resource-errors)
+    - [For beginners](#for-beginners)
+    - [Advanced option on process level](#advanced-option-on-process-level)
+- [Custom configuration](#custom-configuration)
+  - [Updating containers (advanced users)](#updating-containers--advanced-users-)
+  - [nf-core/configs](#nf-core-configs)
+  * [Run sentieon](#run-sentieon)
+  * [Azure Resource Requests](#azure-resource-requests)
+  * [Running in the background](#running-in-the-background)
+  * [Nextflow memory requirements](#nextflow-memory-requirements)
+
+# Introduction
```

Comment:
```suggestion
## Introduction
```

Reply:
We already have the "main" heading at the top of the document. Same comment on subsequent "main" headings